service/static/index.html

```diff
@@ -0,0 +1,125 @@
+          Create, Retrieve, Update, and Delete an Inventory:
+          <div class="well">
+            <div class="form-horizontal">
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="inventory_id">Inventory ID:</label>
+                <div class="col-sm-6">
+                  <input type="text" class="form-control" id="inventory_id" placeholder="Enter ID of Inventory">
+                </div>
+                <div class="col-sm-4">
+                  <button type="submit" class="btn btn-primary" id="retrieve-btn">Retrieve</button>
+                  <button type="submit" class="btn btn-danger" id="delete-btn">Delete</button>
+                </div>
+              </div>
+            </div> <!-- form div -->
+
+            <div class="form-horizontal">
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="product_id">Product Id:</label>
+                <div class="col-sm-6">
+                  <input type="text" class="form-control" id="product_id" placeholder="Enter the Product Id" type="number" step="1">
+                </div>
+                <div class="col-sm-4">
+                    <button type="submit" class="btn btn-danger" id="disable-btn">Disable</button>
+                  </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="quantity">Quantity:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="quantity" placeholder="Enter the quantity for the Inventory">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="restock_level">Restock Level:</label>
+                <div class="col-sm-10">
+                    <input type="text" class="form-control" id="restock_level" placeholder="Enter the restock level for the Inventory">
+                  </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="condition">Condition:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="condition">
+                        <option value="" selected>Select the condition for the inventory</option>
+                        <option value="new">New</option>
+                        <option value="open_box">Open Box</option>
+                        <option value="used">Used</option>
+                  </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="available">Availability:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="available">
+                        <option value="" selected>Select the availability for the inventory</option>
```

Comment:
Here as well
`<option value="" selected disabled> `

Reply:
we need this option for user to set back to empty value

Reply:
Got it! Thanks!